test/new-e2e/tests/agent-platform/rpm/rpm_test.go

```diff
@@ -31,7 +31,7 @@ var (
 	osVersion    = flag.String("osversion", "", "os version to test")
 	platform     = flag.String("platform", "", "platform to test")
 	architecture = flag.String("arch", "", "architecture to test (x86_64, arm64)")
-	majorVersion = flag.String("major-version", "7", "major version to test (6, 7)")
+	majorVersion = flag.String("major-version", "6", "major version to test (6, 7)")
```

Comment:
<!-- Suggestions propose improvements to the current subject. It’s important to be explicit and clear on what is being suggested and why it is an improvement. -->
**💬 suggestion**
It should also not support version 7, as it is not built by this pipeline